ozaria/site/common/ozariaUtils.js

```diff
@@ -188,3 +190,22 @@ export const getNextLevelLink = (levelData, options) => {
   }
   return link
 }
+
+export function internationalizeLevelConfig (levelConfig, userLocale) {
```

Comment:
It seems this can be used for any object across the site and not necessarily level data, so it might make sense to rename `levelConfig` to a generic name eg: just `config`/`data`/`configObject` etc.

Reply:
Makes sense if we use this for more than level config - updated.